README.md

```diff
@@ -13,95 +13,60 @@
 
 Kubernetes has made it very easy to deploy applications to the cloud at a higher scale than ever, but the development practices have not evolved at the same speed as application deployment patterns.
 
-Today, most developers try to either run parts of the infrastructure locally, or just test these integrations directly in the cluster via CI jobs or the "docker build, docker push, kubectl apply" cycle. It works, but this workflow is painful and incredibly slow.
+Today, most developers try to either run parts of the infrastructure locally or just test these integrations directly in the cluster via CI jobs or the *docker build/redeploy* cycle. It works, but this workflow is painful and incredibly slow.
 
-Okteto makes this cycle a lot faster by launching development environment in Kubernetes. You write your code locally using your favorite tools and Okteto detects the code changes and instantly synchronizes your code to your development environment. There you can use your compilers and hot reloaders to see your changes live in seconds.
+Okteto makes this cycle a lot faster. You write your code locally from your favorite IDE and Okteto detects the code changes and instantly updates your Kubernetes applications. **It works on any cluster, local or remote**.
 
-## Install
+## How it works
 
-MacOS / Linux
-```
-curl https://get.okteto.com -sSfL | sh
-```
+Okteto decouples deployment from development. You can deploy your application with kubectl, Helm, a serverless framework, or even a CI pipeline and use Okteto to develop it. This is especially useful for cloud-native applications where deployment pipelines are not trivial.
 
-Windows
-```
-Download https://downloads.okteto.com/cli/okteto.exe and add it to your $PATH.
-```
+Okteto replaces a running deployment by a development container. You write code from your local IDE and Okteto instantly synchronizes your code changes to your development container. The development container can be any docker image, where you install all the dev tools needed by your application: compilers, debuggers, hot reloaders... Okteto gives you a terminal to your development container to build, test, and run your application as you would from a local terminal. Files are synchronized both ways.
 
-You can also directly download the precompiled binary from [the releases page](https://github.com/okteto/okteto/releases).
+![Okteto](docs/okteto-architecture.png)
```

Comment:
maybe I can update the diagram to use the concepts of "development container" in it, and make some other improvements

Reply:
That would be cool. I am going to merge it, but if you can, update the image later.